xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -439,6 +439,19 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
   if (m_streaminfo)
   {
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_AUDIOOUTPUT_SUPPORTSDTSHDCPUDECODING))
```

Comment:
that's the inevitable consequences if such decisions are taken by non-maintainers and even non-devs. I see the history, you changed it after a comment by @MartijnKaijser who has closed PR https://github.com/xbmc/xbmc/pull/4586 in favor of a fix in ffmpeg. Now we have the change in ffmpeg 4586 but still not working because of the reasons I have given.